src/translations/main_de.ts

```diff
@@ -154,7 +154,7 @@ Möglicherweise ist OpenSSL nicht installiert.</translation>
     <name>DialogAddEditNewEmail</name>
     <message>
         <source>&quot;New Email&quot; Entry</source>
-        <translation>Neue-E-Mail Eintrag</translation>
+        <translation>&quot;Neue E-Mail&quot; Eintrag</translation>
```

Comment:
Subsatntive werden zusammen geschrieben. Besser zu lesen ist vielleicht: Eintrag »Neue E-Mail«

Reply:
Es geht hier um einen Eintrag in dem "Neue E-Mail"-Menü, wobei "Neue E-Mail" der Name des Menüs ist. Dein Vorschlag suggeriert aber einen einzelnen Eintrag namens "Neue E-Mail".

Reply:
Wie macht er das? Diese Schreibweise ist falsch, wird leider oft verwendet: "Neue E-Mail"-Menü. Richtig wäre Neue-E-Mail-Menü oder »Menü „Neue E-Mail“«.

Reply:
Wir müssen uns hier überlegen, wo diese Übersetzung verwendet wird: Im Titel eines Dialogs, der den Benutzer einen neuen Menü-Unterpunkt zum Hauptmenüpunkt "Neue E-Mail" hinzufügten lässt.
![Titel des dialogs](https://user-images.githubusercontent.com/6966049/77566079-717f4400-6ec5-11ea-9bbf-7d8ff0175dfa.png)
Wir müssen hier `"Neue E-Mail"` als ein Wort betrachten. Der Benutzer soll durch die Nennung des exakten Namens sofort erkennen, um welchen Menüpunkt es sich handelt:
![Taskleistenmenü](https://user-images.githubusercontent.com/6966049/77566329-c753ec00-6ec5-11ea-8f06-b1773964c89d.png)
`Menü „Neue E-Mail“` drückt für mich nicht aus, dass ich einen Eintrag des Menüs bearbeite.
`Eintrag »Neue E-Mail«` wird für mich im Kopf zu `Eintrag »Neue E-Mail« bearbeiten`, also den Menüpunkt "Neue E-Mail" im Hauptmenü bearbeiten. Das ist zwar nicht komplett falsch, aber sagt nicht aus, dass der Nutzer nur einen Menü-Unterpunkt des "Neue E-Mail" Menüpunkts bearbeiten.
Ich denke, wenn wir es zu `"Neue E-Mail"-Eintrag` ändern, sollte es auch grammatikalisch korrekt sein, oder?

Reply:
Das stimmt, »Neue E-Mail« ist dort ein Wort und in diesem Fall ist alles ein Wort. Alle drei einzelnen Wörter sind das Substantiv. Richtige Zeichensetzung: Neue-E-Mail-Eintrag

Reply:
Ich denke hier ist eher das Wort »Eintrag« etwas unglücklich. Mir fällt aber gerade leider nichts besseres ein.

Reply:
> »Neue E-Mail« ist dort ein Wort [...] Richtige Zeichensetzung: Neue-E-Mail-Eintrag

Das widerspricht sich doch. Wenn "Neue E-Mail" ein eigenes Wort ist, dann darf ich doch nicht einfach das Wort verändern und ein Bindestrich einfügen. Das würde den Namen verändern, weswegen er ja auch in Anführungszeichen steht. Ich glaube, du behandelst "Neue E-Mail" als zwei Wörter und nicht einen Namen.

Reply:
Der Bindestrich verändert kein Wort, er fügt Worte zu einem zusammen! Ich behandel es schon als eins, deshalb schreibe ich es ja auch zusammen. Im deutschen werden Substantive zusammen geschrieben.

Reply:
In dem Fall handelt es sich doch bei "Neue E-Mail" um einen Namen. Ein Bindestich verändert den Namen.